client/containers/workflow/getters.js

```diff
@@ -48,11 +50,26 @@ const getters = {
       ...item,
       pendingTaskType: PENDING_TASK_TYPE_CHILD_WORKFLOW,
     })),
+  [WORKFLOW_EXECUTION_PENDING_DECISIONS]: state => {
+    const pendingDecision = get(state, 'workflow.execution.pendingDecision');
+
+    if (!pendingDecision) {
+      return [];
+    }
+
+    return [
+      {
+        ...pendingDecision,
+        pendingTaskType: PENDING_TASK_TYPE_DECISION,
+      },
+    ];
+  },
```

Comment:
> pending task filter "Decisions"

Reply:
This data entity is an object or null. This function converts to either an empty array or an array with one object decision if present in the dataset.